include/libopencm3/stm32/h7/usart.h

```diff
@@ -49,12 +47,6 @@
 #define UART7           UART7_BASE
 #define UART8           UART8_BASE
 /**@}*/
-
-BEGIN_DECLS
-
-END_DECLS
-
```

Comment:
there's not really any reason to delete this, you really only need the include change here.